### PR TITLE
Better fake data to help debug without a backend

### PR DESCRIPTION
**Pull request by Hixie**

### CODE REVIEW

**Comment by Hixie** [comment]
Here's what it looks like:
2020-04-03-125811_3600x2160_scrot

File: app_flutter/lib/state/flutter_build.dart [hunk 1]
```diff
@@ -233,14 +233,12 @@ class FlutterBuildState extends ChangeNotifier {
     List<CommitStatus> statuses,
     CommitStatus statusToFind,
   ) {
-    for (int index = 0; index < statuses.length; index++) {
+    for (int index = 0; index < statuses.length; index += 1) {
```
Comment (CaseyHillers) [inline app_flutter/lib/state/flutter_build.dart:236]: Is this left over from debugging?

File: app_flutter/lib/state/flutter_build.dart [hunk 2]
```diff
@@ -304,18 +301,14 @@ class FlutterBuildState extends ChangeNotifier {
 
   /// Assert that there are no duplicate commits in [statuses].
   bool _statusesAreUnique(List<CommitStatus> statuses) {
-    final Map<RootKey, bool> uniqueStatuses = <RootKey, bool>{};
-
-    for (int i = 0; i < statuses.length; i++) {
+    final Set<RootKey> uniqueStatuses = <RootKey>{};
```
Comment (CaseyHillers) [inline app_flutter/lib/state/flutter_build.dart:304]: Thanks for improving it! :)

File: app_flutter/lib/state/flutter_build.dart (hunk 1 above)
Comment (Hixie) [reply app_flutter/lib/state/flutter_build.dart:236]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#prefer-over :-)

**Pull request merged**